src/aat/resources/features/F-143 - Search Case Users V2/S-597.1.td.json

```diff
@@ -0,0 +1,21 @@
+{
+	"title": "when a specific User ID is supplied for a specific case ID, then the case roles relating only to the User ID for that specific Case ID must be returned",
+	"_guid_": "S-597.1",
+	"_extends_": "S-597",
+
+  "operationName": "Search Case-Assigned Users and Roles",
+  "method": "POST",
+  "uri": "/case-users/search",
```

Comment:
Having to repeat this across the new scenarios, since these tests are extending the original `GET` ones.

Reply:
Refactored in 0bafb53e2e68dfb1a645083a25f66bd291764681 and 8a786cb6f86f004458e2c1b5cdd57c12fb5f4d90 (to avoid inherritting queryParams) however both sets of tests still share the same `expectedResponse` values.